The below commit contains only a whitespace change and was backported in
order to avoid future conflicts.

commit 35cfefd96062145eeb8aee6bd72d07e0909a6b2e
Author: Florian Weimer <fweimer@redhat.com>
Date:   Mon Aug 20 14:57:13 2018 +0200

    malloc: Add ChangeLog for accidentally committed change
    
    Commit b90ddd08f6dd688e651df9ee89ca3a69ff88cd0c ("malloc: Additional
    checks for unsorted bin integrity I.") was committed without a
    whitespace fix, so it is adjusted here as well.

diff --git a/malloc/malloc.c b/malloc/malloc.c
index c07463001a65af90..eb6a8ff33c0c313b 100644
--- a/malloc/malloc.c
+++ b/malloc/malloc.c
@@ -3745,7 +3745,7 @@ _int_malloc (mstate av, size_t bytes)
           if (__glibc_unlikely (bck->fd != victim)
               || __glibc_unlikely (victim->fd != unsorted_chunks (av)))
             malloc_printerr ("malloc(): unsorted double linked list corrupted");
-          if (__glibc_unlikely (prev_inuse(next)))
+          if (__glibc_unlikely (prev_inuse (next)))
             malloc_printerr ("malloc(): invalid next->prev_inuse (unsorted)");
 
           /*
